### DISCUSSION TITLE
Warning of lost edit changes

### DISCUSSION THREAD

**Discussion by juanpgarza** [OCA/odoo-community.org#91] [Q&A] [answered]
I am seeing that up to Odoo 14 when modifications were made to a form and then the discard button was selected it shows this warning message "The record has been modified. Your changes will be discarded. Do you wish to continue?"

This message is no longer shown as of Odoo 15. Does anyone know of a module that returns this functionality?

**Comment by legalsylvain** [comment, marked as answer]
- Yes, in recent version there is an "autosave" mechanism + no more warning
- AFAIK, there is no OCA module that restore the previous Odoo UX.
- if the module is proposed, I think it will be placed in this repo : https://github.com/OCA/web. 

CC : @OCA/web-maintainers
